### PR TITLE
Tweaked Descriptions – Part 4

### PR DESCRIPTION
**Pull request by 4Luke4**

### CODE REVIEW

File: _opcodes/op143-bgee.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Moreover, unlike <a href="#op111">opcode #111</a>, this opcode will not "equip" the item that's created in this particular slot.
+{% endcapture %}
+
+{% include important.html %}
+
+{% capture note %}
+When using a non-permanent timing mode, it applies a delayed <a href="../_opcodes/op123.html">opcode #123</a> effect based on the duration, but with an empty resource field, resulting in nothing being removed.
+{% endcapture %}
+
+{% include note.html %}
```
Comment (lynxlynxlynx) [inline _opcodes/op143-bgee.html]: Why the separate file? Looks like the extra info should be in the main opcode.

File: _opcodes/op152-bgee.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+			<li>OUTRO – Closing the Book</li>
+			<li>HEXXATDR – Dragomyr's Crypt (Hexxat)</li>
+			<li>DORNGORG – Resurrection Gorge (Dorn)</li>
+			<li>DORNGORG – Ritual Circle (Dorn)</li>
+			<li>NEERAES – Town Bell (Neera)</li>
+			<li>RASAADMI – Sharran (Rasaad)</li>
+			<li>RASAADTE – Temple (Rasaad)</li>
+		</ul>
+	</li>
+</ul>
```
Comment (lynxlynxlynx) [inline _opcodes/op152-bgee.html]: I suggest you add this to the bg2 one and convert the lists to a table, one column per game. Will keep everything shorter.

File: _opcodes/op163-bgee.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Does NOT remove "Increment" or "Set % Value" effects, nor any using the alternate movement rate opcode, <a href="../_opcodes/op176.html">opcode #176</a>.
+{% endcapture %}
+
+{% include important.html %}
+
+{% capture note %}
+Similarly affects <a href="../_opcodes/op177.html">opcode #177</a>/<a href="../_opcodes/op283-bgee.html">opcode #283</a> effects if the <a href="../file_formats/ie_formats/eff_v2.htm">EFF</a> file is using the above values.
+{% endcapture %}
+
+{% include note.html %}
```
Comment (lynxlynxlynx) [inline _opcodes/op163-bgee.html]: This is likely also true for the other games, so please add a note to the main one. Eg. "Also check out the EE description here."

File: _opcodes/op165-bgee.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	<li>Their scripts stop running.</li>
+</ul>
+
+<br/>This effect sets <a href="../files/ids/bgee/stats.htm#70">STAT #70</a> to 1.
+
+{% capture note %}
+Both parameters should be 0.
+{% endcapture %}
+
+{% include important.html %}
```
Comment (lynxlynxlynx) [inline _opcodes/op165-bgee.html]: This should be in the main one-

File: _opcodes/op168-bgee.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+	<li>This effect removes the creature from the player's party (if they were in it).</li>
+	<li>The effect will keep the creature out of the party and any other Area for its duration, but does not re-instate the creature once it expires.</li>
+	<li>They can still be recovered through script actions once the effect expires.</li>
+</ul>
+
+{% capture note %}
+Both parameters should be 0 (zero).
+{% endcapture %}
+
+{% include important.html %}
```
Comment (lynxlynxlynx) [inline _opcodes/op168-bgee.html]: Move to main.

File: _opcodes/op242-bgee.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Removes STATE_CONFUSION and <a href="../_opcodes/op128.html">opcode #128</a> effects from the targeted creature(s).
```
Comment (lynxlynxlynx) [inline _opcodes/op242-bgee.html]: Good catch, the original is suspect.

File: _opcodes/op143-bgee.html (hunk 1 above)
Comment (4Luke4) [reply _opcodes/op143-bgee.html]: Moved.

File: _opcodes/op152-bgee.html (hunk 2 above)
Comment (4Luke4) [reply _opcodes/op152-bgee.html]: Done.

File: _opcodes/op163-bgee.html (hunk 3 above)
Comment (4Luke4) [reply _opcodes/op163-bgee.html]: Done.

File: _opcodes/op165-bgee.html (hunk 4 above)
Comment (4Luke4) [reply _opcodes/op165-bgee.html]: Moved.

File: _opcodes/op168-bgee.html (hunk 5 above)
Comment (4Luke4) [reply _opcodes/op168-bgee.html]: Done.

File: _opcodes/op163.html [hunk 7]
```diff
@@ -10,4 +10,5 @@
 iwd2: 1
 pst: 1
 ---
-Applies the free action effect to the targeted creature(s), restoring the creatures movement rate to its default value if the current movement rate value is below the default for the animation type.
+Applies the free action effect to the targeted creature(s), restoring the creatures movement rate to its default value if the current movement rate value is below the default for the animation type.<br/>
+Also check out the EE description <a href="https://gibberlings3.github.io/iesdp/opcodes/bgee.htm#op163">here</a>.
```
Comment (lynxlynxlynx) [inline _opcodes/op163.html]:
```suggestion
Also check out the EE description <a href="bgee.htm#op163">here</a>.
```

Comment (4Luke4) [reply _opcodes/op163.html]: Done.

**Comment by 4Luke4** [comment]
@lynxlynxlynx 

No Travis check this time?

File: _opcodes/op165-bgee.html (hunk 4 above)
Comment (lynxlynxlynx) [reply _opcodes/op165-bgee.html]: You forgot to remove this file though.

File: _opcodes/op143-bgee.html (hunk 1 above)
Comment (lynxlynxlynx) [reply _opcodes/op143-bgee.html]: Is the extra file for bgee still needed then?

Comment (4Luke4) [reply _opcodes/op143-bgee.html]: Removed.

File: _opcodes/op165-bgee.html (hunk 4 above)
Comment (4Luke4) [reply _opcodes/op165-bgee.html]: Removed.

**Comment by lynxlynxlynx** [comment]
In progress:
https://travis-ci.org/github/Gibberlings3/iesdp/builds/667674582?utm_medium=notification&utm_source=github_status

**Comment by 4Luke4** [comment]
> In progress:
> https://travis-ci.org/github/Gibberlings3/iesdp/builds/667674582?utm_medium=notification&utm_source=github_status

Unlike past times, there's no visual feedback on GitHub (yellow point / red cross / green tick)

**Comment by 4Luke4** [comment]
> Looks fine, thanks!

You're welcome!

What's the issue with the note of[ opcode #331](https://gibberlings3.github.io/iesdp/opcodes/bgee.htm#op320)? Why is it displayed like that?

**Comment by lynxlynxlynx** [comment]
It's interpreted as markdown syntax for tables (|) ...

**Comment by 4Luke4** [comment]
> It's interpreted as markdown syntax for tables (|) ...

So what's the issue exactly? The previous version didn't have any `|` character and the table was there too...

**Comment by lynxlynxlynx** [comment]
Square brackets are special too, so perhaps that contributes.

**Comment by 4Luke4** [comment]
> Square brackets are special too, so perhaps that contributes.

OK, I'll try to quote those characters then. Can I quote them with `\`? If not, which is the correct symbol?

**Comment by lynxlynxlynx** [comment]
Just use normal parentheses or skip the numbers, since you link to the ids anyway.

**Pull request merged**